src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/ExceptionUtils.vb

```diff
@@ -27,6 +178,23 @@ Namespace Microsoft.VisualBasic.CompilerServices
             Return VbMakeExceptionEx(vbErrors.ObjNotSet, GetResourceString(SR.ID91)) ' 91 - ObjNotSet
         End Function
 
+        Friend Shared Function VbMakeException(ByVal hr As Integer) As System.Exception
+            Dim sMsg As String
+
+            If hr > 0 AndAlso hr <= &HFFFFI Then
+                sMsg = GetResourceString(CType(hr, vbErrors))
+            Else
+                sMsg = ""
+            End If
+            VbMakeException = VbMakeExceptionEx(hr, sMsg)
+        End Function
+
+        Friend Shared Function VbMakeException(ByVal ex As Exception, ByVal hr As Integer) As System.Exception
+            ' UNDONE - Err() requires port of Information.vb, ProjectData.vb
```

Comment:
Undone here.

Reply:
Meaning "TODO" rather than something that was reverted?

Reply:
Yes, these are as yet unported  classes.